.github/workflows/build-test.yaml

```diff
@@ -108,6 +108,21 @@ jobs:
       - name: Build crate
         run: cargo build --target ${{ matrix.target }} --all-features
 
+  msrv:
+    name: MSRV
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust 1.46.0
```

Comment:
Could you use a global env variable of msrv?
You can add `env` workflow between `on` and `jobs` workflow:
Ref: https://docs.github.com/en/actions/learn-github-actions/environment-variables#about-environment-variables

```yml
env:
  msrv: 1.46.0
```

Then, you can reference the version by `${{ env.msrv }}`.